src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Unit.Tests/ComInterfaceGenerator.Unit.Tests.csproj

```diff
@@ -19,6 +19,12 @@
             Link="Common\CustomStructMarshallingCodeSnippets.cs" />
     <Compile Include="..\Common\ICustomMarshallingSignatureTestProvider.cs"
             Link="ICustomMarshallingSignatureTestProvider.cs" />
+    <Compile Include="..\LibraryImportGenerator.UnitTests\Verifiers\CSharpAnalyzerVerifier.cs"
+            Link="Verifiers\CSharpAnalyzerVerifier.cs"/>
+    <Compile Include="..\LibraryImportGenerator.UnitTests\Verifiers\CSharpCodeFixVerifier.cs"
+            Link="Verifiers\CSharpCodeFixVerifier.cs"/>
+    <Compile Include="..\LibraryImportGenerator.UnitTests\Verifiers\CSharpVerifierHelper.cs"
+            Link="Verifiers\CSharpVerifierHelper.cs"/>
```

Comment:
Let's move these 3 files to the `Common` folder that `TestUtils.cs` is in now that we're using them from multiple unit test projects.